bin/wmagent-resource-control

```diff
@@ -109,10 +109,11 @@ def getCMSSiteInfo(pattern):
     for seName in siteDB.cmsNametoList(pattern, "SE", "result.json"):
         cmsNames = siteDB.seToCMSName(seName)
```

Comment:
I think this misses cmsNames FNALDisk some how.
https://cmsweb.cern.ch/sitedb/data/prod/site-names
We should create the function directly maps psn - pnn. using above call. I think still site db need to be updated. I will check with Diego and Nicolo.